spec/features/projects_manage_spec.rb

```diff
@@ -141,7 +141,9 @@
 
     context "import & Export" do
       before do
-        project.stories.create(title: "php upgrade", description: "quick php upgrade")
+        project.stories.create(title: "php upgrade", description: "quick php upgrade", status: :approved)
+        project.stories.create(title: "pending story", description: "pending stories don't export", status: :pending)
+        project.stories.create(title: "rejected story", description: "rejected stories don't export", status: :rejected)
```

Comment:
Didn't you think we should update those expectations? 

https://github.com/fastruby/points/blob/7b52e9feca45cf3832b60f7376614a5f6e39c70d/spec/features/projects_manage_spec.rb#L161

https://github.com/fastruby/points/blob/7b52e9feca45cf3832b60f7376614a5f6e39c70d/spec/features/projects_manage_spec.rb#L162

Reply:
Thank you for the feedback. To avoid confusion or false positives I moved the new stories to only be created inside of the `export` spec and not in the before_all block.